contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,82 @@
+Suites may change over time with additions, deletions, and modifications whenever
+the SIG learns more about the problem space and adapts the implementations.
+Hence, Validation Suites are not subject to the deprecation cycles such as those 
+that apply to features and modification decisions are left to the decision making
+provess(es) of the SIG.  The end users of the Validation Suites are strongly 
+encouraged to participate in the upkeep and maintenance of the suite so
+they know what changes are coming up and can adapt their 
+product/implementation to the updates of the test suite.
+
+## Running Validation Suite Tests
+
+Validation Suite tests are designed to be run even when there is no cloud provider
+configured. The [Testing Commons](https://github.com/kubernetes/community/tree/master/sig-testing) 
+subproject within the Kubernetes sig-testing
+community is currently tasked with adding a framework for writing test suites and
+this new framework may be used for running Validation Suites. This new framework is 
+optional because there is not a prescribed approach for how a Validation Suite needs to 
+be run.  Instead, the authors of the Validation Suite need to ensure that the 
+mechanisms for running the Validation Suite are easy to use, provide sufficient
+documentation, and follow patterns similar to existing e2e tests. 
+In addition, the mechanism should provide the  ability to select some tests and 
+skip others.  
+
+### Validation Suite Test Comment Metadata
+
+Each Validation suite test must include the following piece of metadata
+within its associated comment:
+
+- `Release`: indicates the Kubernetes release that the test was added to the
+  Validation test suite. If the test was modified in subsequent releases
+  then those releases should be included as well (comma separated)
+- `Testname`: a human readable short name of the test
+- `Description`: a detailed description of the test. This field must describe
+  the required behaviour of the Kubernetes components being tested using
+  [RFC2119](https://tools.ietf.org/html/rfc2119) keywords. This field
+  is meant to be a "specification" of the tested Kubernetes features, as
+  such, it must be detailed enough so that readers can fully understand
+  the aspects of Kubernetes that are being tested without having to read
+  the test's code directly. Additionally, this test should provide a clear
+  distinction between the parts of the test that are there for the purpose
+  of validating Kubernetes rather than simply infrastructure logic that
+  is necessary to setup, or clean up, the test.
+
+### Sample Validation Suite Test
+
+The following snippet of code shows a sample Storage Validation Suite test's 
+metadata.  Note how the test contains a bracketed label with name of the Validation
+Suite that this test belongs to. 
+
+```
+/*
+  Release : v1.14
+  Testname: CSI Attach, Non-attachable volume used in a Pod
```

Comment:
nit: We might need to be more specific about the substructure of the name (if any). e.g. Is 'CSI Attach' a category of some sort?  Why is 'Non' capitalized?  Is this intended to be a proper sentence? e.g. "CSI-Attach (used as a verb) a non-attachable volume (object) used in a Pod (qualifying clause).

In summary, perhaps this example nname is a bad one, unless you explain why it's pretty weird-looking.

Reply:
nit: PS: Why not just 'Name' instead of 'Testname'?

Reply:
@quinton-hoole-2 the above was a verbatim sample taken from one test in a new validation suite that @brahmaroutu added. I believe he is keeping the syntax consistent with conformance tests to make promotion simpler.

Reply:
@quinton-hoole-2 We use this same mechanism on Conformance Suite, a TestName is currently parsed by a tool that can generate conformance documentation, we can use the same tool here.  
You are right that the Test name has a category that defines the what is being tests ex: PodSpec or CSI Attach, etc. The description part is a free form text and need no be capitalized.
We can change the Example if it is confusing WDYT?

Reply:
@brahmaroutu I think just put in a link to the documentation explaining the weird formatting of the test name.